huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go

```diff
@@ -0,0 +1,582 @@
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// API: DataArtsStudio POST /v2/{project_id}/design/standards
+// API: DataArtsStudio GET /v2/{project_id}/design/standards
+// API: DataArtsStudio PUT /v2/{project_id}/design/standards/{id}
+// API: DataArtsStudio GET //v2/{project_id}/design/standards/{id}
+// API: DataArtsStudio DELETE /v2/{project_id}/design/standards
+func ResourceDataStandard() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDataStandardCreate,
+		UpdateContext: resourceDataStandardUpdate,
+		ReadContext:   resourceDataStandardRead,
+		DeleteContext: resourceDataStandardDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceDataStandardImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"workspace_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the workspace ID of DataArts Studio.`,
+			},
+			"directory_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the directory ID that the data standard belongs to.`,
+			},
+			"values": {
+				Type:        schema.TypeSet,
+				Elem:        dataStandardValueSchema(),
+				Required:    true,
+				Description: `Specifies the value of data standard attributes.`,
+			},
+			"directory_path": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the path of the directory.`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the status of the data standard.`,
+			},
+			"created_by": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the name of creator.`,
+			},
+			"updated_by": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the name of updater.`,
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the creation time of the data standard.`,
+			},
+			"updated_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the latest update time of the data standard.`,
+			},
+			"new_biz": {
+				Type:        schema.TypeList,
+				Elem:        dataStandardNewBizSchema(),
+				Computed:    true,
+				Description: `Indicates the biz info of manager.`,
+			},
+		},
+	}
+}
+
+func dataStandardValueSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"fd_name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the name of the data standard attribute.`,
+			},
```

Comment:
This parameter is required in the API request body
The document ditto.

Reply:
done